### ISSUE TITLE
Auto theming makes the dark mode pale

### ISSUE THREAD

**Issue by MW-J97** [ssborbis/ContextSearch-web-ext#691]
When I enable the auto theming, it makes the dark theme pale. 
I don't know actually which one is the normal one. It also may be affected by Vivaldi new feature of dark mode. 
![BeFunky-collage](https://github.com/ssborbis/ContextSearch-web-ext/assets/73688266/7125f165-9f25-4087-94ff-a0aa52bd1a5b)

**Comment by ssborbis** [comment]
Something appears to be changing the colors on the CS logo at the top left. That's weird. I don't have any code that could be doing that. To be honest, neither of the screenshots you posted look exactly like the menu should. Something else must be going on.

![image](https://github.com/ssborbis/ContextSearch-web-ext/assets/24310997/e895aa1f-d08e-4301-9b41-bebb3a7c0756)

**Comment by MW-J97** [comment]
> CS logo at the top left.

For this, it's due to the dark mode of Vivaldi. But, the dark theme itself of CS is not related to the dark mode of Vivaldi.

**Comment by ssborbis** [comment]
It appears to be the Force setting in vivaldi's dark setting

![image](https://github.com/ssborbis/ContextSearch-web-ext/assets/24310997/dedbbfc7-27aa-470f-9adc-29a55e643906)

**Comment by MW-J97** [comment]
> It appears to be the Force setting in vivaldi's dark setting

Yes, as you said, but I actually want this option enabled. So, I switched the CS theme to light and with “Force a dark theme” enabled, it changes the CS settings page to dark.
